routes/outbound-lims.xml

```diff
@@ -8,21 +8,27 @@
                 <simple>${body.tableName} == 'test_order'</simple>
                 <setProperty name="lab-order-uuid">
```

Comment:
Wondering why we are using here an Exchange Property instead of a Message Header to hold this info. Is there any preference of one over the other?

Reply:
Few researches done now, but best conclusion I've found is:

> The headers and properties in Apache Camel framework are sometimes used interchangeably to communicate values between the processes within a single route, but when we want to carry data across different routes that is when the behaviours start to differ. Headers can be lost at endpoints basically as they usually represent some component-specific things.

So, are we going to be using this info in another route? If so, then it's likely a good place to keep it as Exchange Property.

Reply:
Yes, we are re-using the properties across the routes.